profile_bluesky/startup/15-spec-config.py

```diff
@@ -4,52 +4,23 @@
-ti3y = EpicsMotor('8idi:m16', name='ti3y', labels=("motor",))
 tth = EpicsMotor('8idi:sm1', name='tth', labels=("motor",))
-ta1_x = EpicsMotor('8ida:TA1:x', name='ta1_x', labels=("motor",))
-ta1_z = EpicsMotor('8ida:TA1:y', name='ta1_z', labels=("motor",))
-ta2_x = EpicsMotor('8ida:TA2:x', name='ta2_x', labels=("motor",))
-ta2_z = EpicsMotor('8ida:TA2:y', name='ta2_z', labels=("motor",))
-ta2fine = EpicsMotor('8ida:sm9', name='ta2fine', labels=("motor",))
-diamx = EpicsMotor('8idd:m1', name='diamx', labels=("motor",))
-diamz = EpicsMotor('8idd:m2', name='diamz', labels=("motor",))
-ti2_x = EpicsMotor('8idi:TI2:x', name='ti2_x', labels=("motor",))
-ti2_z = EpicsMotor('8idi:TI2:y', name='ti2_z', labels=("motor",))
-ti3_x = EpicsMotor('8idi:TI3:x', name='ti3_x', labels=("motor",))
-ti3_z = EpicsMotor('8idi:TI3:y', name='ti3_z', labels=("motor",))
-ti1_x = EpicsMotor('8idi:TI1:x', name='ti1_x', labels=("motor",))
-ti1_z = EpicsMotor('8idi:TI1:y', name='ti1_z', labels=("motor",))
 pind1z = EpicsMotor('8idi:m3', name='pind1z', labels=("motor",))
-shuttz = EpicsMotor('8idi:m2', name='shuttz', labels=("motor",))
-bewinx = EpicsMotor('8idi:m17', name='bewinx', labels=("motor",))
-bewinz = EpicsMotor('8idi:m11', name='bewinz', labels=("motor",))
 pind2z = EpicsMotor('8idi:m4', name='pind2z', labels=("motor",))
-samx = EpicsMotor('8idi:m54', name='samx', labels=("motor",))
-samy = EpicsMotor('8idi:m49', name='samy', labels=("motor",))
-samz = EpicsMotor('8idi:m50', name='samz', labels=("motor",))
-samth = EpicsMotor('8idi:m51', name='samth', labels=("motor",))
-sampit = EpicsMotor('8idi:m52', name='sampit', labels=("motor",))
-samchi = EpicsMotor('8idi:m53', name='samchi', labels=("motor",))
 tth_act = EpicsMotor('8idi:m63', name='tth_act', labels=("motor",))
 bstop = EpicsMotor('8idi:m27', name='bstop', labels=("motor",))
-ti4_x = EpicsMotor('8idi:TI4:x', name='ti4_x', labels=("motor",))
-ti4_z = EpicsMotor('8idi:TI4:y', name='ti4_z', labels=("motor",))
-shuttx = EpicsMotor('8idi:m1', name='shuttx', labels=("motor",))
-ccdz = EpicsMotor('8idi:m91', name='ccdz', labels=("motor",))
 alpha = EpicsMotor('8idi:sm2', name='alpha', labels=("motor",))
-ti4zu = EpicsMotor('8idi:m30', name='ti4zu', labels=("motor",))
-ti4zdo = EpicsMotor('8idi:m31', name='ti4zdo', labels=("motor",))
-ti4zdi = EpicsMotor('8idi:m32', name='ti4zdi', labels=("motor",))
-ti4xu = EpicsMotor('8idi:m28', name='ti4xu', labels=("motor",))
-ti4xd = EpicsMotor('8idi:m29', name='ti4xd', labels=("motor",))
 tthAPD = EpicsMotor('8idi:sm3', name='tthAPD', labels=("motor",))
-ccdx = EpicsMotor('8idi:m90', name='ccdx', labels=("motor",))
-fccdx = EpicsMotor('8idi:m25', name='fccdx', labels=("motor",))
-fccdz = EpicsMotor('8idi:m83', name='fccdz', labels=("motor",))
 foceye = EpicsMotor('8idi:m37', name='foceye', labels=("motor",))
-piezox = EpicsMotor('8idi:m69', name='piezox', labels=("motor",))
-piezoz = EpicsMotor('8idi:m70', name='piezoz', labels=("motor",))
+
+# What is si6?
```

Comment:
si6: Slit 6 in 8-ID-I station.  During discussions, we identified this slit is no longer in use.